### DISCUSSION TITLE
[Feature Request] Time-Based Passcode AutoLock Options for iOS

### DISCUSSION THREAD

**Discussion by fredrickflinstone** [standardnotes/forum#3490] [Suggestions]
There are two options for Passcode AutoLock on iOS: 

1) Immediately
2) On Quit

Can you please add an option to do “1 Minute”? (like the Windows Desktop app has)

Reason being, I use the Passcode lock (in addition to Biometrics) as a safety net, for if my phone was ever stolen. 

Having to enter my passcode every time I swap back to the app (within a few seconds) is not preferred. Selecting “On Quit” is also not preferred because I will forget to quit it.

The option to set a time-based passcode AutoLock, like the Windows Desktop app has, would allow me to relax better swapping between my notes and other apps.

Thank you for your consideration. I am a new 5 year plan member and really finding the app valuable so far!

**Comment by korg91** [comment]
This is a very important feature, not only for Passcode Lock but **for Biometrics Lock too** (I'm on Android). In my experience, the _On Quit_ option will leave the app unlocked for a very long time (hours) unless the app is manually closed. On the other hand, the _Immediately_ option is way too short -- just switching to another app for a few seconds to copy some text and then back to SN will trigger the autolock, which seems really unnecessary and affects the workflow.

Several security-oriented apps support custom timeout options for autolock, e.g. Signal.
